source/com/athaydes/specks/assertion/assertions.ceylon

```diff
@@ -47,16 +48,26 @@ shared AssertionResult expectCondition(Boolean expectedToBeTrue)
 shared String platformIndependentName(Type<Throwable>|Throwable exception) =>
         exception.string.replace("::", ".");
 
+"A value to signal \"no comparison should be done\" in an assertion"
+shared object noCheck {}
```

Comment:
The name here is misleading. From the code, I can see that this is just a flag object to indicate no assertion should be done on the `Throwable`'s message. So the obvious name would be `noErrorMessageCheck`.